CMake compilation failure of tango-9-lts

$ cmake --version
cmake version 2.8.11
$ cmake .. -DOMNI_BASE=~/packages/omniORB-4.2.1 -DZMQ_BASE=~packages/zeromq-4.0.7 -DCMAKE_BUILD_TYPE=RELEASE -DCMAKE_VERBOSE_MAKEFILE=true -DIDL_BASE=~/software/tango-idl
-- The C compiler identification is GNU 4.8.5
-- The CXX compiler identification is GNU 4.8.5
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
System TANGO_HOST=tcfidell11:10000
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.27.1")
-- Found Doxygen: /usr/bin/doxygen (found version "1.8.5")
Using OMNI_BASE=~/packages/omniORB-4.2.1
Using ZMQ_BASE=~packages/zeromq-4.0.7
Verifying ~/packages/omniORB-4.2.1/bin/omniidl
found GNU compiler ...
setup for release build ...
CMake Error at configure/CMakeLists.txt:70 (add_compile_options):
  Unknown CMake command "add_compile_options".
Call Stack (most recent call first):
  CMakeLists.txt:18 (include)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 2.8.12)
 include(CTest)
 
 enable_testing()

